.devcontainer/Dockerfile

```diff
@@ -3,7 +3,6 @@
 ################################################################
 FROM ghcr.io/magma/magma/bazel-base:latest as devcontainer
 
-ARG HOME=/home/vscode
```

Comment:
I decided to duplicate `/home/vscode`. it's used in two places, and it's not important that those two places share the same prefix: The python environment belongs to the root user and could live anywhere.